bump_version.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# bump_version.sh (show|major|minor|patch|prerelease|build)
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+VERSION_FILE=src/example/_version.py
```

Comment:
This should reflect the version file in this project.

Reply:
Good catch @mcdonnnj

Reply:
Fix in 6e4c3db